sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<h2>Train the XGBoost Model  </h2><p>Finally we launch a training job to train the XGBoost model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
done